src/scenes/Event/components/Leaderboard/index.js

```diff
@@ -30,19 +30,19 @@ const tabs = [messages.mostQTUM, messages.mostBOT, messages.biggestWinner];
 @observer
 export default class Leaderboard extends React.Component {
   handleNext = () => {
-    this.props.store.eventPage.activeStep = this.props.store.eventPage.activeStep + 1;
+    this.props.xstore.activeStep = this.props.xstore.activeStep + 1;
```

Comment:
dont pass the eventStore. just reference the `eventStore` from the injected AppStore here.